Applications/Spire/Source/OrderImbalanceIndicator/FilteredOrderImbalanceIndicatorModel.cpp

```diff
@@ -17,42 +18,37 @@ std::tuple<boost::signals2::connection,
       const boost::posix_time::ptime& start,
       const boost::posix_time::ptime& end,
       const OrderImbalanceSignal::slot_type& slot) {
-  auto signal = std::make_unique<FilteredOrderImbalanceSignalConnection>(
-    FilteredOrderImbalanceSignalConnection{
-      {OrderImbalanceSignal(), start, end}, {}});
+  auto signal = std::make_unique<OrderImbalanceSignalConnection>(
+    OrderImbalanceSignalConnection{OrderImbalanceSignal(), start, end});
   auto callback = [this, signal = signal.get()] (auto& imbalance) {
       if(is_imbalance_accepted(imbalance)) {
```

Comment:
Fix indentation.

Reply:
Should the closing brace line up with the declaration?

Reply:
auto callback = [this, signal = signal.get()] (auto& imbalance) {
      if(is_imbalance_accepted(imbalance)) {
        signal->m_imbalance_signal(imbalance);
      }
    };